python/sglang/srt/managers/scheduler_output_processor_mixin.py

```diff
@@ -861,6 +867,28 @@ def stream_output_generation(
 
                 retraction_counts.append(req.retraction_count)
 
+                queue_times.append(req.time_stats.get_queueing_time())
+                forward_entry_times.append(req.time_stats.forward_entry_time)
+
+                if req.time_stats.prefill_start_time > 0.0:
+                    prefill_delays.append(
```

Comment:
This is redundant code. It appeared twice in this file. You should move all common code to `class TimeStats`